canvas/published_date.py

```diff
@@ -58,7 +58,7 @@ def published_date_resp_parsing(self, response):
         try:
             audit_events = json.loads(response.result().text)
         except JSONDecodeError as e:
-            logger.info(f"Error in parsing the response {e.message}")
+            logger.info(f"Error in parsing the response {e.msg}")
```

Comment:
I guess this was a bug you wanted to fix? Might want to just make that change in a tiny separate PR, in case it comes up in production.

Reply:
I just submitted the PR

Reply:
Yep, looks like we're good. I will re-build with this after the current job finishes, so the morning job will have it.